docs/DeveloperGuide.md

```diff
@@ -320,8 +320,7 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 ### Glossary
```

Comment:
Maybe good to have a few more terms for the glossary? 
Could include client, insurance agent.
Can remove later unnecessary terms later.